css/style.css

```diff
@@ -0,0 +1,752 @@
+
+.head-about {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding: 20px;
+  padding-bottom: 30px;
+  background-image: url("../images/mobile_intro_bg.png");
+  background-repeat: no-repeat;
+  background-position: center;
+  text-align: center;
+  color: #ec5242;
+}
+
+.about-top {
+  font-size: 20px;
+  padding-top: 50px;
+  text-transform: uppercase;
+}
+
+.about-title {
+  font-family: 'COCOGOOSE', sans-serif;
+  font-size: 36px;
+  font-style: normal;
+  font-weight: normal;
+  line-height: 50px;
+  padding-top: 25px;
+}
+
+.about-p {
+  background-color: #fff;
+  border: 1px solid #e4f1f1;
+  font-size: 20px;
+  color: #333;
+  padding: 20px;
+}
+
+.about-contact {
+  font-size: 16px;
+  color: #333;
+}
+
+.about-email {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: #333;
+  font-size: 22px;
+  font-weight: 600;
+  text-decoration-color: #ec5242;
+  text-underline-position: under;
+}
+
+.about-marathon {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  text-align: center;
+  padding: 30px;
+}
+
+.about-marathon-title {
+  font-size: 28px;
+}
+
+.about-marathon-info {
+  font-size: 20px;
+}
+
+.about-marathon-img {
+  height: 50%;
+}
+
+.about-summits {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  gap: 30px;
+  padding: 20px;
+  padding-bottom: 30px;
+  background-color: #e4f1f1;
+}
+
+.about-summits .about-summit-title {
+  font-size: 28px;
+}
+
+.about-summits .about-summit-info {
+  font-size: 16px;
+}
+
+.about-summits .about-summit-info-img1,
+.about-summits .about-summit-info-img2 {
+  background-size: 100%;
+  height: 15rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 38px;
+  color: #fff;
+}
+
+.about-summits .about-summit-info-img1 {
+  margin-bottom: 4rem;
+  background-image: linear-gradient(45deg, rgba(235, 59, 59, 0.5), rgba(251, 78, 78, 0.5)), url("../images/about1.jpg");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.about-summits .about-summit-info-img2 {
+  background-image: linear-gradient(45deg, rgba(235, 59, 59, 0.5), rgba(251, 78, 78, 0.5)), url(../images/about3.jpg);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.partner {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding: 30px;
+  background-color: #333;
+  color: #fff;
+  text-align: center;
+}
+
+.partner .partner-title {
+  font-size: 22px;
+  color: #e9ecef;
+}
+
+.partner .partners-list {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  gap: 30px;
+  justify-content: center;
+  list-style: none;
+}
+
+.partner .partners-list .column {
+  float: left;
+}
+
+.end {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  gap: 30px;
+  font-size: 12px;
+  padding: 14px 14px;
+}
+
+.end .logo {
+  width: 130px;
+}
+
+@media only screen and (min-width: 768px) {
```

Comment:
- [ ] On the desktop version, the about me page design does not show up the partners section, and the footer has a dark background. You should implement those changes for the desktop version only! (the mobile one looks like the original).

Original design:
![image](https://user-images.githubusercontent.com/66967274/147249406-1a5d5fd5-aae5-4f28-84ed-7ff4a42f28f0.png)

Your design:
![image](https://user-images.githubusercontent.com/66967274/147249534-29a392c3-22c6-43b4-b71c-d27a63a8c050.png)

Reply:
updated